pkg/test/runner/runner.go

```diff
@@ -311,7 +311,10 @@ func (r *Runner) compareResult(exitErr error, stdout string, stderr string, tmpP
 func (r *Runner) compareOutput(stdout string, stderr string) error {
 	expectedStderr := r.testCase.Config.StdErr
 	if expectedStderr == "" && stderr != "" {
-		return fmt.Errorf("unexpected stderr %s", stderr)
+		// special case for fn-eval/invalid-fn-config-file
```

Comment:
You can change the content of invalid config file in `fn-eval/invalid-fn-config-file` to what you prefer.

Reply:
The problem here is that we can't get the stderr from the config file when it is invalid, so there needs to be a special case for it. 

Why do we have invalid-fn-config-file? Is it to test the harness itself? Is it necessary to keep it?

Reply:
What is invalid is the function config file not test config file. So I am not sure why can't we get the stderr here.

Reply:
ah, my mistake. I misread the files and thought that was the test config file. I'll fix that, thanks!